protocol/contracts/mock/MockHybridPool.sol

```diff
@@ -0,0 +1,55 @@
+
+pragma solidity ^0.5.17;
+
+import "../oracle/pool/HybridPoolBase.sol";
+pragma experimental ABIEncoderV2;
+
+contract MockHybridPool is HybridPoolBase {
+    address _gold;
+    uint256 _goldReserve;
+    uint256 _backingAssetReserve;
+    bool _mockLocalReserves;
```

Comment:
If we manually set the reserves via `setReserves`, the return those values, otherwise delegate to the UNIV2 passed in.